src/status_im/contact/subs.cljs

```diff
@@ -128,3 +128,28 @@
                acc))
            {}
            contacts)))
+
+(re-frame/reg-sub
+ ::dapps
+ (fn [db]
+   (:contacts/dapps db)))
```

Comment:
minor nitpick - is there any value in having this as a local sub as opposed to just nesting `(:contacts/dapps db)` in the calling code?

Reply:
No personal opinion on this. I just reverted some previously deleted code.